packages/clickable/stories/clickable.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
```

Comment:
Can we use `React from "react";` like we normally do?

Reply:
Yes, we can at some point I thought we needed to do that.

Reply:
I found out why we need it. If we do not import complete react the prop documentation is not created.